TestOne.gocd.yaml

```diff
@@ -134,4 +149,63 @@ pipelines:
               - docker
               tasks:
               - script: |
-                  echo command to run smoke test 
\ No newline at end of file
+                  echo command to run smoke test
+
+  CapacityTesting:
+    group: AT20Group
+    label_template: ${COUNT}
+    lock_behavior: none
+    display_order: -1
+    environment_variables:
+      GIT_COMMIT_HASH: 1.2
+      TARGET_HOST: 192.168.0.21
+    materials:
+      TestOne:
+        pipeline: TestOne
```

Comment:
For the next PR, please update the material to use from the previous pipeline. If the acceptance testing passes, then continue with the capacity